components/ai_chat/core/common/mojom/tab_informer.mojom

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2025 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// You can obtain one at https://mozilla.org/MPL/2.0/.
+
+module ai_chat.mojom;
+
+import "url/mojom/url.mojom";
+
+struct Tab {
+    int32 id;
+    int32 content_id;
+    string title;
+    url.mojom.Url url;
+};
+
+interface TabListener {
+    // If we find we need it, we can make this more granular
+    TabsChanged(array<Tab> windows);
+};
+
+interface TabInformer {
```

Comment:
`TabInformerService` (TabDataTracker per dm) and the actual class should be `TabInformerServiceImpl`

Reply:
I've used `TabInformerService` as the actual class, which extends `mojom::TabInformerService`.